src/test/esperanza/finalizationstate_tests.cpp

```diff
@@ -103,7 +106,8 @@ BOOST_AUTO_TEST_CASE(initialize_epoch_insta_justify) {
 // 6% per year given that the total deposit of validator is 150Mln units
 BOOST_AUTO_TEST_CASE(initialize_epoch_reward_factor) {
 
-  FinalizationStateSpy spy;
+  finalization::Params params = finalization::Params::TestNet();
```

Comment:
Because some of these tests were written to examine testnet parameters, see corresponding change in finalizationstate_utils file:
```C++
const FinalizationParams params = CreateChainParams(CBaseChainParams::TESTNET)->GetFinalization();
```
 I kept default constructed params in places which testnet-independent.